src/conlink/core.cljs

```diff
@@ -906,7 +906,7 @@ General Options:
      docker-eth0-addresses (when docker-eth0? (intf-ipv4-addresses "eth0"))
      _ (swap! ctx merge {:kmod-ovs? kmod-ovs?
                          :kmod-mirred? kmod-mirred?
-                         :docker-eth0? docker-eth0?
+                         :docker-eth0? (or docker-eth0? show-config)
```

Comment:
A little white lie :-P.  I assume having `show-config` only show unenriched configs didn't satisfy your needs in the case where you ran into this (perhaps adding a new flag to show enriched/derived configs, which could then do all the same checks and fatal exits as a normal conlink startup)? I can imagine you wanted to see the enriched configs, and most of the ideas I can think of to separate config building/enrichment from validation really hurt clarity/locality.

If we wanted to give this scenario a name but leave all checks/error handling as is, we could add a `no-checks`/`no-fatal` key to the `opts` that get passed into 'enrich-network-config and set it when `show-config` is set. 🤷 Otherwise, I'm good here - just brainstorming.

Reply:
Yeah, the enriched configs are the thing that I want printed by --show-config. In particular, `enrich-network-config` does the merging of the links that have the same container and interface name. And yep, I did go down the path of passing a `checked?` options through the call chain. But it was messy and a lot of the checks that call fatal should still be fatal because they are checking the sanity of the config itself. So yes, this is a bit of a hack, but it was much simpler so I went with this for now.